lib/lotus/mailer/dsl.rb

```diff
@@ -98,6 +98,122 @@ def templates(value = nil)
         end
       end
 
+      # When a value is given, specify the sender of the email
+      # Otherwise, it returns the sender of the email
+      #
+      # @param value [Object] String or Proc to be evaluated containing the sender of the email
+      #
+      # @return [String] the sender of the email
+      #
+      # @since 0.1.0
+      #
+      # @example With String
+      # class StringMailer
+      #   include Lotus::Mailer
+      #
+      #   from "noreply@example.com"
+      #
+      # end
+      #
+      # @example With Procs
+      # class ProcMailer
+      #   include Lotus::Mailer
+      #
+      #   from = Proc.new { customized_sender }
+      #
+      #   def customized_sender
+      #     "user_sender@example.com"
+      #   end
+      # end
+      def from (value = nil)
+        if value.nil?
+          return new.eval_proc(@from)
```

Comment:
Don't we have `else` already? You can remove this explicit `return` for the sake of Ruby idiom